dockerfiles/azure-repos/Dockerfile

```diff
@@ -56,6 +56,9 @@ ENV PORT 7341
 # This will be used as the webhook payload URL
 ENV BROKER_CLIENT_URL "https://<broker.client.hostname>:$PORT"
 
+# the url of your snyk git client (including scheme and port)
+# ENV GIT_CLIENT_URL https://<snyk-git-client-host>:<snyk-git-client-port>
```

Comment:
question: Why is this commented out? These images are built and deployed to dockerhub, right?

Reply:
I see that below some Dockerfiles actually also have `ENV BROKER_CLIENT_URL` commented out.
Not that it really makes a difference (those are runtime ENVs which will have be provided with `-e` anyway).
More a question of consistency I guess.

Reply:
Yeah, I guess it's commented because this is a non-standard env variables and is only required in certain cases at the moment thus it needs to be passed explicitly.

Reply:
I honestly didn't know why some env were ending up in the dockerfile while others don't and still others end up in the dockerfile but commented out. @pkey explanation above seems sound enough tho.

Reply:
But does it make sense to have them there? Should we just add this to README.md instead? Customer does not really look at those `Dockerfile`s anyways, he just cares about thing he needs to pass to `docker run`

@ohad2712 @carlos-snyk

Reply:
I think commented out env vars are useless and don't need to be there. Broker users have the `.env.sample` files already to see what env vars are being used and can change